src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {
+
+  },
+});
+
```

Comment:
- [ ] After creating your reducers, you're expected to import them in this file and use them inside the reducer object with their key and value pairs. 

`reducer: { books: booksReducer, categories: categoriesReducer }`

- [ ] Kindly import the bookReducer and the categoriesReducer and make use of them inside the reducer object in your store 👍🏽